File: app/src/main/java/com/intoverflown/pos/ui/splash/SplashSuccessActivity.java
package com.intoverflown.pos.ui.splash;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import com.bumptech.glide.Glide;
import com.intoverflown.pos.R;
import com.intoverflown.pos.databinding.ActivitySplashSuccessBinding;

public class SplashSuccessActivity extends AppCompatActivity {

    private ActivitySplashSuccessBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivitySplashSuccessBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());

        Glide.with(this).asGif().load(R.drawable.check).error(R.drawable.error_img).into(binding.gifCheck);
    }
}

File: app/src/main/java/com/intoverflown/pos/ui/splash/SplashSuccessActivity.kt
package com.intoverflown.pos.ui.splash

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.intoverflown.pos.databinding.ActivitySplashSuccessBinding

class SplashSuccessActivity : AppCompatActivity() {

    private var binding : ActivitySplashSuccessBinding? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivitySplashSuccessBinding.inflate(layoutInflater)
        setContentView(binding!!.root)
    }
}